modules/installation-ibm-cloud-tested-machine-types.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.adoc
+
+[id="installation-ibm-cloud-tested-machine-types_{context}"]
```

Comment:
```suggestion
:_mod-docs-content-type: REFERENCE
[id="installation-ibm-cloud-tested-machine-types_{context}"]
```
please add the "reference" content type to each of the tested machine types modules right before the `id` line.